test/binaries/test-postgre.sql-9.4.3.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+int main() {
+	printf("This program is designed to test the cve-bin-tool checker.");
+	printf("It outputs a few strings normally associated with postgresql-9.4.3");
+	printf("They appear below this line.");
+	printf("------------------");
+	printf("PostgreSQL 9.4.3");
```

Comment:
This is testing the same "contains" code as test-postgres.sql.9.3.6.c and won't increase coverage, so one of these can be removed.

Reply:
I have made the required changes.